roles/oc_setup/README.md

```diff
@@ -19,6 +18,17 @@ Access to a valid kubeconfig file via an `KUBECONFIG` environment variable.
 export KUBECONFIG=<kubeconfig_path>
 ```
 
+## Users
+
+These are the users created and the roles associated to them. See [official documentation about the default roles](https://docs.openshift.com/container-platform/4.17/post_installation_configuration/preparing-for-users.html#default-roles_post-install-preparing-for-users)
```

Comment:
Now that docs.openshift.com would be decommisioned in Nov, maybe it's worth start using docs.redhat.com in our doc references, wdyt? the redirection would still work
https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/postinstallation_configuration/post-install-preparing-for-users

Reply:
Ahh, great point I'll make the change here

``` suggestion
## Users

These are the users created and the roles associated to them. See [official documentation about the default roles](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/postinstallation_configuration/post-install-preparing-for-users#default-roles_post-install-preparing-for-users)
```